### PR TITLE
Add content for new full workshop - Prepare your data using Python and VS Code

### PR DESCRIPTION
**Pull request by ornelladotcom**
## Use this template to add a workshop to this repo. Add the following items:

- [x] Your workshop design document, filled in with some details about your proposed workshop including its title and the three stages in the designer delineated, using the `workshop-designer.md` file in the template.

- [x] A `README.md` file that describes the steps of the workshop. 

- [x] Powerpoint Slides for the workshop.

- [x] The workshop's solution in the `/solution` folder.

- [ ] An embedded Train the Trainer video.

- [x] A Call to Action to enhance the workshop with a direct line to Microsoft Learn by linking to an appropriate Learn module.

A reviewer will be assigned to the workshop and will work with you to shape it for production. Thank you for your contribution!

### CODE REVIEW

**Review by jlooper — COMMENTED** [review]
hi, looking good! Are you able to generate slides? We can keep this in draft if you need a bit more time.

**Comment by ornelladotcom** [comment]
@jlooper i'll add them tonight or tomorrow morning! thanks for your review on the other stuff :)

File: prepare-data-python/README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+So far, you've learned how to use pandas methods to examine some aspects of a DataFrame, and fill in, remove, and combine data. The final way we will seek to understand our data is by [creating visualizations](https://docs.microsoft.com/learn/modules/data-manipulate-clean/6-exploratory-statistics-visualization?WT.mc_id=academic-55190-ornella).
+
+## Next steps
+
+- [Explore and analyze data with Python](https://docs.microsoft.com/learn/modules/explore-analyze-data-with-python/?WT.mc_id=academic-55190-ornella)
+- [Introduction to machine learning
+](https://docs.microsoft.com/learn/modules/introduction-to-machine-learning/?WT.mc_id=academic-55190-ornella)
+- [Discover the role of Python in space exploration](https://docs.microsoft.com/learn/paths/introduction-python-space-exploration-nasa/?WT.mc_id=academic-55190-ornella)
+
+## Optional Transfer Knowledge activity
```
Comment (jlooper) [inline prepare-data-python/README.md:74]: Let's call this "Practice" (I've standardized the terminology to make it more friendly)

**Review by jlooper — COMMENTED** [review]
this is looking great! Do you want to include a 'solution' folder with a notebook to show everything? Up to you if so.

File: prepare-data-python/README.md (hunk 1 above)
Comment (ornelladotcom) [reply prepare-data-python/README.md]: fixed!

**Comment by ornelladotcom** [comment]
Just added the slides and the solution!

**Review by jlooper — APPROVED** [review]
great! thanks!

**Pull request merged**